### PR TITLE
The Religion Patch

### PR DESCRIPTION
**Pull request by GalacticLiaison**
Rough Unofficial Patch Notes

# Beta_v1.2.0

## Religion

Elf Holy Sites will now be created at Expedition sites you complete three trials at
NEW! Spark Syncretism Tenet
Now have a chance to encounter a couple Valar (Gods) at expeditions sites after the portal has been opened

## Story

New event that allows player to move portal pieces to their capital so they can restore the Grand Portal there.

## Traditions

NEW! Blessing: Endless Lord
- First born child born is an exact genetic copy of their parent lord.
- Other children born normally
- Child receives all standard genetic traits of their lord parent.
- Elf traits treated differently, first born can still ascend higher than parent
- Child receives a new trait that tracks how many iterations of that lord there has been

NEW! Blessing: Beguiling Nature
- Characters gain Lustful after 3 seducing three lords and upgrade it to Passionate after seducing 5 lords.
- Unlocks new **Entrance** Scheme
- Characters with the Seductress Lifestyle trait will upgrade it to Enchantress after 3 successful Entrance Schemes. 

NEW! Tradition: Aeluran Etiquette
- Increases county religion and culture conversion speeds.

## Introducing Valar Blessing!

NEW! Blessing of Vána
- Gift Of Many Mothers: The people of this culture will give birth to six girls for every one boy born
- +3 stewardship; -3 martial

NEW! Blessing of Tulkas Astaldo
- Gift Of Endless Armies: The people of this culture will give birth to six boys for every one girl born
- +3 martial; -3 stewardship;

## Ascension

The remainder of the Elf traits have been added to the game bringing the total levels of Ascension to 12!
Players can not Ascend past High Elf until after the portal has been opened.
Players can not yet Ascend to the highest tier of traits
Elf Traits
- Now give Elves huge fertility penalties that increase in severity as you Ascend.
- Now give no prowess loss from age
- Now will randomly appear on characters of Elven culture

## Magic

Started to dip the mod's toes into magic
NEW! Magi Lifestyle trait
NEW! genetic magic ability traits!
    - Spark Gifted
    - Spark Savant
    - Spark Prodigy
NEW! Entrance scheme unlocked by "Beguiling Nature" blessing. This is the first official "spell" of the mod.
    - Make target slightly "obsessed" with you via your Spark powered aura through "chance" encounters at events like Feasts.
    - grants a perpetual hook on target if successful
    - odds of success are lowered if trying to Entrance a character of same elf tier. Impossible to entrance character of higher elf tier without the new Enchantress Trait
NEW! Enchantress trait
    - First trait of a series of planned traits that represent a type of Magi specialization.
    - requires the Seductress Lifestyle trait to unlock

## Aelurans

Added annual logic to replace dead Matriarch/Matrons/Sisters
- This is a very simplistic solution that will be expanded upon in the future with proper elections and what not.
Added event to take magically talented daughters of Elf Lords to be trained at the Aeluran's HQ at age 7
Added event where they return as trained Aeluran Sister Magi at age 15
Aeluran Advisor
- Magistration:
    - Now has chance to gain a perpetual basic hook called "Religious Exaltation" on a courtier or vassal
    - Now has a chance to recruit a new Magi to join your court
    - Now has a chance to receive a unit of special NEW! MAAs "Aeluran Magi" sent from the Order
- Indoctrinate
    - now has small chance of increasing control in selected county
    - now has small chance of converting culture in selected county
- Undermine
    - now has small chance of converting county to your faith
    - now has a very small chance of converting enemy heir to your faith

## Misc

Elf Beautification
- removed giant like head change for Daelurin bloodline trait
- NEW! Game Rule: apply or remove prowess muscle gain effect for female characters
- NEW! Game Rule: apply or remove makeup effect from Beauty traits

## Bug Fixes/Tweaks

MAA
- The Nerf Hammer has returned!
    - Some stats reduced
    - Elf Units will now cost significantly more to balance how strong they are

Adventurer Trait
- was too OP giving minus 50% MAA maintenance right off the bat for just going on a single expedition. Now must have fully leveled adventurer to get that bonus

Tradition: Adventurer Guilds
    - now allows potentially two trials per run

Tradition: Familial Familiarity
    - now properly unlocks family consanguinity

Tradition: Tribal Ascension
    - fixed loss of succession options
    - fixed loss of feudal contracts

Tradition: The Rut
    - Rut was a bit too frequent happening every year. Changing to occur once a decade

Tradition: Familial Familiarity
    - Accidentally disabled Purer/Purest-Blooded inheritance, should be working again

Elven Adaptability innovation now removes reform cooldown for culture

Elf Religion Names = Elf Culture Names

Accidentally permanently unlocked "challenge rival to a fight" character interaction. Fixed

Portal pieces are now removed after restoring the Grand Portal

Added "OK" dismiss button to tutorial window

Basically the mod makes several faith tenets disappear in faith reformations, mostly ones added in the Persia update. Selection of existing faiths that have the doctrines is fine, but I don’t think new player-created faith have the option to select such tenets

### CODE REVIEW

**Comment by GalacticLiaison** [comment]
PATCH IS LIVE! 🥳 Merging!
